sdk/core/azure-core/azure/core/pipeline/policies/_universal.py

```diff
@@ -339,7 +339,7 @@ class HttpLoggingPolicy(SansIOHTTPPolicy):
     """The Pipeline policy that handles logging of HTTP requests and responses.
     """
 
-    DEFAULT_HEADERS_WHITELIST = set([
+    DEFAULT_HEADERS_ALLOWLIST = set([
```

Comment:
I would suggest don't do this in this PR.

Reply:
Based on a conversation with @xiangyan99, I'm going to revert these changes in this PR and open another PR for these changes in core. Issue filed here https://github.com/Azure/azure-sdk-for-python/issues/12219